### ISSUE TITLE
Fix Jcrop.gif 404s

### ISSUE THREAD

**Issue by akash1810** [guardian/grid#807]
After bundling, the JCrop.gif image 404s at the following path.

An example request:

```
2015-06-11 11:39:21,849 - [INFO] - from request in play-akka.actor.default-dispatcher-661
77.91.250.206 - "GET /images/jspm_packages/github/tapmodo/Jcrop@0.9.12/css/Jcrop.gif HTTP/1.1" 404 1246 "https://media.gutools.co.uk/images/fe5cfc16a305aecf0e16837bb594bac7d864d06f" 1ms
```

This is causing errors and PagerDuty alarms.

We should look at how we bundle the CSS and/or how Jcrop pulls this file in so that this doesn't happen.

**Comment by theefer** [comment]
See https://github.com/systemjs/plugin-css/issues/52

**Comment by theefer** [comment]
Fixed in a really pathetic way by #1007
